bot/exts/moderation/clean.py

```diff
@@ -627,12 +627,18 @@ async def clean_cancel(self, ctx: Context) -> None:
     @command()
     async def purge(self, ctx: Context, users: Greedy[User], age: Optional[Union[Age, ISODateTime]] = None) -> None:
         """
-        Clean messages of users from all public channels up to a certain message age (10 minutes by default).
+        Clean messages of `users` from all public channels up to a certain message `age` (10 minutes by default).
 
-        The age is *exclusive*, meaning that `10s` won't delete a message exactly 10 seconds old.
+        Requires 1 or more users to be specified. For channel-based cleaning, use `clean` instead.
+
+        The `age` is *exclusive*, meaning that `10s` won't delete a message exactly 10 seconds old.
```

Comment:
I don't think this would really matter in practice. A message being *exactly* n seconds long is unlikely given snowflakes have millisecond precision (which I assume is being used?).

Also note that in the help command everything is in italics already, so "exclusive" is actually excluded from being in italics.

Reply:
nvm that was there already 😅

Reply:
I could also remove that original last line? I agree that it's kind of not needed as it's unlikely that kind of precision matters in a typed command.

Also in the demo image you can see the `* *` causing the weird italic font behavior.

Reply:
Updated in 87de12c

```diff
- The `age` is *exclusive*, meaning that `10s` won't delete a message exactly 10 seconds old.
+ `age` can be a duration or an ISO 8601 timestamp.
```